code/provision/playbooks/group_vars/tessella/k8s-vault.yml

```diff
@@ -1,822 +1,738 @@
 $ANSIBLE_VAULT;1.1;AES256
```

Comment:
Without decrypting, is the only change to this file to update the Test SSL cert?

Reply:
Yes - just looks a lot shorter as the previous certificate was a chain - but when testing only the new cert itself was required.